src/nni_manager/core/nnimanager.ts

```diff
@@ -116,6 +116,10 @@ class NNIManager implements Manager {
         await this.storeExperimentProfile();
         this.log.debug('Setup tuner...');
 
+        if(expParams.multiPhase && this.trainingService.isMultiPhaseJobSupported) {
+            this.trainingService.setClusterMetadata('multiPhase', expParams.multiPhase.toString());
```

Comment:
Should we add this for resume experiment?

Reply:
Yes, updated